source/delegate/package.json

```diff
@@ -27,7 +27,7 @@
   },
   "dependencies": {
     "@airswap/indexer": "0.4.2",
-    "@airswap/swap": "2.3.2",
+    "@airswap/swap": "2.3.3",
```

Comment:
do we need to increment delegate as its dependency has updated.

Reply:
That's not how hwe've done it in previous PRs. We normally just bump the thing that's changed.
e.g. Ethan's PR this morning https://github.com/airswap/airswap-protocols/pull/319/files

I do see your point though.

Reply:
Bumped delegate and therefore delegate factory